AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/Keyrings/RawAESKeyring.dfy

```diff
@@ -386,7 +386,7 @@ module RawAESKeyring {
       //# If no decryption succeeds, the keyring MUST fail and MUST NOT modify
       //# the [decryption materials](structures.md#decryption-materials).
       return Failure(Types.CollectionOfErrors(list := errors,
-                                              message := "Raw AES Keyring was unable to decrypt any encrypted data key. The list of encountered Exceptions is avaible via `list`."));
+                                              message := "Raw AES Keyring was unable to decrypt any encrypted data key. The list of encountered Exceptions is available via `list`."));
```

Comment:
before merging in; ask @texastony about this. We are technically changing the error message and that could be a breaking change...